app/views/dit_landing_page/_page_header.html.erb

```diff
@@ -15,19 +15,26 @@
 
 <header class="dit-landing-page__header--with-bg-image">
   <div class="dit-landing-page__header-text-block">
-    <div class="govuk-width-container landing-page_header-text-container">
+    <div class="govuk-width-container">
       <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-grid-column-two-thirds dit-landing-page_header-text-container <%= 'dit-landing-page_header-text-container--no-translation-nav' unless defined?(presented_taxon) %>">
+          <% if defined?(presented_taxon) %>
+            <%= render "govuk_publishing_components/components/translation-nav", {
+              translations: presented_taxon.translation_links,
+              inverse: true,
+            } %>
+          <% end %>
+
           <h1 class="dit-landing-page__page_title">
-            <%= t("dit_landing_page.page_header") %>
+            <%= sanitize(t("dit_landing_page.page_header")) %>
```

Comment:
Minor: how come we need to sanitize now? Ah, maybe it's because of the `&nbsp`?

Reply:
Yes, we use an HTML entity in the page_header so I'm using sanitize to escape it otherwise it will get rendered as text.